docker/openproblems-r-extras/Dockerfile

```diff
@@ -1,19 +1,29 @@
 FROM singlecellopenproblems/openproblems-r-base:latest
 
+
+RUN apt-get update
+RUN apt-get install -y libhdf5-dev
```

Comment:
put this in the same docker command, and after `apt-get update` delete the list repo to minimize docker image size. See recent sagemaker commit from Dan.